layouts/index.html

```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
       <h5 class="mt-0">Chat with us:</h5>
         <a href="https://matrix.to/#/#invidious:matrix.org"><img alt="Matrix" src="https://img.shields.io/matrix/invidious:matrix.org?label=Matrix&color=darkgreen"></a>
         <a href="https://web.libera.chat/?channel=#invidious"><img alt="Libera.chat (IRC)" src="https://img.shields.io/badge/IRC%20%28Libera.chat%29-%23invidious-darkgreen"></a>
+
+        <br/>
```

Comment:
Why here?

Reply:
Imo, It's a bit too cramped without a buffer between the heading section and the feature overview.

With buffer:
![w buffer](https://user-images.githubusercontent.com/70992037/132124001-59939227-6428-4920-9794-5d19936fb0a8.png)

Without buffer:
![no buffer](https://user-images.githubusercontent.com/70992037/132124003-e425aa82-d577-4726-b9b3-111ecf221825.png)

Reply:
True.